src/user-management/user-management.ts

```diff
@@ -1,97 +1,52 @@
-import { WorkOS } from '../workos';
+import { sealData, unsealData } from 'iron-session';
+import { createRemoteJWKSet, decodeJwt, jwtVerify } from 'jose';
+import { fetchAndDeserialize } from '../common/utils/fetch-and-deserialize';
```

Comment:
Looks like these got auto-sorted. I _can_ remove if it's too noisy, but we should probably have them sorted anyway.